README.md

```diff
@@ -154,6 +117,56 @@ This runs several multi-process tests:
    [RFC6962 entrypoints](https://tools.ietf.org/html/rfc6962#section-4).
 
 
+Working on the Code
+-------------------
+
+Developers who want to make changes to the Trillian codebase need some
+additional dependencies and tools, described in the following sections.  The
+[Travis configuration](.travis.yml) for the codebase is also useful reference
+for the required tools and scripts, as it may be more up-to-date than this
+document.
+
+### Running Codebase Checks
+
+The [`scripts/presubmit.sh`](scripts/presubmit.sh) script runs various tools
+and tests over the codebase.
+
+```console
+# Install various codebase checkers
+go get -u github.com/client9/misspell/cmd/misspell
+go get -u github.com/fzipp/gocyclo
+go get -u github.com/gordonklaus/ineffassign
+go get -u github.com/golang/lint/golint
+# Run checks, tests and code generation
+./scripts/presubmit.sh
```

Comment:
The `presumbit.sh` script regenerates the protos & mocks, it might be worth re-ordering these two sections ("running codebase checks" and "rebuilding generated code") and adding something to say you'll need to follow the instructions in the "rebuilding generated code" section to install the necessary bits.